src/routes/common/logging/logging.service.ts

```diff
@@ -16,26 +16,31 @@ export class RequestScopedLoggingService implements ILoggingService {
     private readonly cls: ClsService,
   ) {}
 
-  info(message: string, ...optionalParams: unknown[]) {
-    this.logger.info(this.transformMessage(message), ...optionalParams);
+  info(message: string | unknown) {
+    this.logger.log('info', this.formatMessage(message));
   }
 
-  error(message: string, ...optionalParams: unknown[]) {
-    this.logger.error(this.transformMessage(message), ...optionalParams);
+  error(message: string | unknown) {
+    this.logger.log('error', this.formatMessage(message));
   }
 
-  warn(message: string, ...optionalParams: unknown[]) {
-    this.logger.warn(this.transformMessage(message), ...optionalParams);
+  warn(message: string | unknown) {
+    this.logger.log('warn', this.formatMessage(message));
   }
 
-  debug(message: string, ...optionalParams: unknown[]) {
-    this.logger.debug(this.transformMessage(message), ...optionalParams);
+  debug(message: string | unknown) {
+    this.logger.log('debug', this.formatMessage(message));
   }
 
-  private transformMessage(message: string): string {
+  private formatMessage(message: string | unknown): unknown {
```

Comment:
(Related to [this comment](https://github.com/safe-global/safe-gelato-relay-service/pull/72#discussion_r1172161986)) Do you think it makes sense to type the return type here as well?

Reply:
Done [172452d](https://github.com/safe-global/safe-gelato-relay-service/pull/72/commits/172452d77b89af1a0f35b8fbfe51aa9b0635b26d)

Removed the `unknown` cast 👍 Now it returns the prototype.

Reply:
I was thinking in using concrete types for these functions return values, like:
```javascript
export interface LogMessage {
  message: string | unknown;
  request_id: string;
  timestamp: string;
}

export interface RouteLogMessage {
  client_ip: string;
  method: string;
  response_time_ms: number;
  route: string;
  path: string;
  status_code: number;
  detail: string | null;
}
```

Any reason why you prefer to not use types here and just return an `Object`? Maybe I'm missing some implication 🙂

Reply:
The object returned is the type itself (the one that you represented). But since this type/prototype was only used in a limited scope I think it's ok to not extract it (trying to balance between verbose declarations) – previously it was doing a cast to `unknown` so that had to change.

I can extract the types though! no strong opinion on this one 👍

Reply:
Oh ok, now I get it 🙂. It's ok the way it is, I leave this up to you. Thanks for the explanation!